src/lua/httpc.lua

```diff
@@ -228,6 +241,56 @@ local function process_headers(headers)
     return headers
 end
 
+-- In the Extended BNF notation of RFC 1341, a Content-Type header field value
+-- is defined as follows: Content-Type := type "/" subtype *[";" parameter].
+-- See RFC 1341, https://www.w3.org/Protocols/rfc1341/4_Content-Type.html
+-- Function splits content-type header for part with type and subtype and part
+-- parameter and returns the first one.
```

Comment:
> -- Function splits content-type header for part with type and subtype and part
> -- parameter and returns the first one.

At first glance I read 'the first one' as `type`, not `type/subtype`. Maybe it worth to spin for a while on the wording.

Reply:
It seems confusing in general. I think you should just state that the function retrieves `type/subtype`. Information about parameters is excessive as we throw them away anyway.